conf/st2.conf.sample

```diff
@@ -100,6 +100,16 @@ logging = conf/logging.exporter.conf
 # Directory to dump data to.
 dump_dir = /opt/stackstorm/exports/
 
+[garbagecollector]
```

Comment:
I'm confused, why would we bundle it with st2common? This has nothing to do with st2common (besides depending on it like many other services). It's a separate standalone service,

Maybe I still don't understand new packaging / current state of the new packaging, but I thought we will have a package per service (or the very least, to start with a package per component).